qp_klp/tests/data/configuration_profiles/iseq_metagenomic.json

```diff
@@ -3,6 +3,25 @@
     "instrument_type": "iseq",
     "assay_type": "Metagenomic",
     "configuration": {
+      "tell-seq": {
+        "integrate_cores": "1",
+        "integrate_mem_limit": "8",
+        "integrate_script_path": "/home/qiita_test/qiita-spots/ADDITIONAL_REQUIREMENTS/integrate-indices-np.py",
```

Comment:
Would this be an issue for other testing? I mean, like testing in a laptop or in barnacle?

Reply:
Nope. It's just meant to reflect what an actual value might look like. it's not actually used in a test. In fact it would be better if I made it less specific.